src/ts/components/core/combobox/TagsInput.tsx

```diff
@@ -80,7 +82,6 @@ const TagsInput = (props: Props) => {
             data-dash-is-loading={
                 (loading_state && loading_state.is_loading) || undefined
             }
-            wrapperProps={{ autoComplete: "off" }}
```

Comment:
Are you fixing a bug by removing `wrapperProps` here? If so should we add a test?

Reply:
When you add the `autoComplete` like this it has no effect because it applies it to the outer container and not the  input field.